api/model-server.yaml

```diff
@@ -539,6 +539,12 @@ paths:
           $ref: '#/components/responses/200'
         "404":
           $ref: '#/components/responses/404'
+  /metrics:
+    get:
+      operationId: getMetrics
+      responses:
+        "200":
+          $ref: '#/components/responses/200'
```

Comment:
Not sure if we should include that in the spec here. This is a different concern then our core content API.

Reply:
There is no real structure whatsoever. I would decide this when dealing with https://issues.modelix.org/issue/MODELIX-683

Reply:
If we don't add the route now, we don't have a breaking change later on ;-) The information here is in any case close to nothing.

Reply:
So you vote to not use the openAPI for the metric endpoint? I would not want to do that. If not: What is your preference here?

Reply:
I would say the whole metrics stuff is a separate concern. And we don't even have the format under our control. I'd see that as orthogonal, maybe with an additional spec if we really want to. In the end, the whole metrics reporting format is standardized by someone else and we can just refer to that spec.

Reply:
I will add a dedicated yaml for this for now. We can use that as a basis for MODELIX-683 and decide then how to proceed further.